site/docs/webrtc/voice-iw.md

```diff
@@ -104,13 +104,13 @@ This sends the payload back to the Voice API
     });
 ```
 
-There’s a limitation to what can be done with Transfer: the fact that there is a single call recognized by the PV platform means that changes to that call will impact the WebRTC interconnection. Once the transfer is complete, you can redirect the call to another point using the [TransferComplete](https://dev.bandwidth.com/voice/bxml/callbacks/transferComplete.html) callback, however this will result in the removal of the WebRTC leg of the call. To do this, you need to specify the transferCompleteUrl attribute. The above example can be found in [Hello-world repository](https://github.com/Bandwidth-Samples/webrtc-hello-world-ts).
+There’s a limitation to what can be done with Transfer: the fact that only a single call recognized by the PV platform means that changes to that call will impact the WebRTC interconnection. Once the transfer is complete, you can redirect the call to another destination using the [TransferComplete](https://dev.bandwidth.com/voice/bxml/callbacks/transferComplete.html) callback, however this will result in the removal of the WebRTC leg of the call. To do this, you need to specify the transferCompleteUrl attribute. The above example can be found in [Hello-world repository](https://github.com/Bandwidth-Samples/webrtc-hello-world-ts).
```

Comment:
```suggestion
There’s a limitation to what can be done with Transfer: the fact that only a single call recognized by the PV platform means that changes to that call will impact the WebRTC interconnection. Once the transfer is complete, you can redirect the call to another destination using the [TransferComplete](../voice/webhooks/transferComplete) callback, however this will result in the removal of the WebRTC leg of the call. To do this, you need to specify the transferCompleteUrl attribute. The above example can be found in [Hello-world repository](https://github.com/Bandwidth-Samples/webrtc-hello-world-ts).
```

Reply:
The `TransferComplete` link currently links to the old site.

Reply:
Updated, but let me confirm with the build before you attempt approval.

Reply:
I believe you updated to the wrong link in your change (BXML Transfer vs the TransferComplete webhook). When a suggestion is made in GitHub you can hit the "Commit suggestion" button to add it automatically.

Reply:
You were too quick - I had not yet completed the submission and checked the build.  Sorry 'bout that - it is good now on the most recent build.

Reply:
after discussion on slack - removed the link as superfluous